plugin-tester-java/build.gradle

```diff
@@ -3,24 +3,27 @@ plugins {
   id 'com.lightbend.akka.grpc.gradle'
 }
 
+def scalaVersion = org.gradle.util.VersionNumber.parse(System.getenv("TRAVIS_SCALA_VERSION") ?: "2.12")
+def scalaBinaryVersion = "${scalaVersion.major}.${scalaVersion.minor}"
+
 akkaGrpc {
   generateClient = true
   generateServer = true
   serverPowerApis = true // This isn't default
+  includeStdTypes = true
+  grpcVersion = "1.32.1"
```

Comment:
Is this now mandatory to specify here? I'd rather see it get inherited from the plugin.

Reply:
no, it is not. the `includeStdTypes`  is false by default as I want to keep existing behaviour